[pull] primary from flybywiresim:primary (#90)
Valastiri/fbw-userguide: docs/beginner-guide/takeoff-climb-cruise.md

I will do this as the very last step -->

### 1. Lineup
**Situation:**

- ATC (Ground or Tower)  has instructed us to hold at a runway holding point
  and wait until we are cleared to "line up" or "take off".<br/>
- Aircraft is still in TAXI state (see previous chapters) and parking brakes
  are set.

![Holding at runway holding point](../assets/takeoff-climb-cruise/Holding.png "Aircraft holding at runway holding point")

Typically, it is here at the latest that we are asked to switch to Tower ATC
frequency for take off clearance.

While approaching the runway holding point or at the latest at the runway
holding point the **"Before take off checklist"** needs to be completed.

!!! info "Before take off checklist"
    ![Before Take Off Checklist](../assets/takeoff-climb-cruise/BeforeTakeOffChecklist.png "Before Take Off Checklist"){ align=right width=180 }

    The "Before Take Off" checklist is divided into two parts:
    
    - **"Down to the line"** (or "Above the line") means **before** "ATC Take Off Clearance".
    - **"Below the line"** means **after** T.O clearance (when lined up) but 
      before starting the roll.

<p/>

> #### Preparation and "Down to the line" checklist pre-T.O-clearance

Reviewer: I saw T.O when looking at NOSE light (no second .) but forgot to change everything. 
It must be consistent. 
And thinking about it I do think it looks better with 2 . (T.O.)

After ATC (Tower) gives clearance to "line up" or "take off" we are allowed
to enter the runway.

"Line up" means that we roll onto the runway and stop at our starting point.
We __MUST__ wait for ATC to give us "take off clearance" before we can continue.

"Cleared for take off" means we are allowed to actually start the take off
when aligned with the runway.

Shortly before we start our take off roll we do the following steps:

- Check PACKS as required<br/>
  (some airlines take off with Packs OFF to allow more power to thrust
  and save fuel - not necessarily required)

- Turn on landing lights and check if Strobe light is in AUTO or ON

    ![landing_lights](../assets/takeoff-climb-cruise/Landing_lights.png){ width=360 }

      The correct switch settings are:
    
      - RWY TURN OFF lt is ON
      - NOSE light is at T.O. (take off)
      - LAND lights are both ON
      - STROBE is on ON or AUTO
      - BEACON, NAV & LOGO should have been on during TAXI already
      - WING is usually only ON for wing inspection and to detect ice 
        accretion on the wing

    !!! info "Lights at Take Off"
        Setting the RWY TURN OFF light to ON, the LAND LIGHTS to ON and

>         the NOSE light to T.O minimizes bird strike hazard during takeoff.

Reviewer 2: Oh and consider for other ones that are tabbed under bullets. Saw a couple but they could also work as is.

>         the NOSE light to T.O minimizes bird strike hazard during takeoff.

Reviewer: These where indented on purpose as they belonged to the bullet point above. Will double check them anyway. If you think they should all be left aligned it is also ok for me.

- Check ENG MODE SEL as required (should be on MODE NORM)

- Set TCAS to TA or TA/TR and traffic to All or ABV)

**A typical standard take off follows these steps:**

!!! info "Airline SOPs"
    Some airline's SOPs (standard operating procedures) might have a different order for these steps.

1. Release parking brake and hold down manual brakes.

- Apply thrust slowly to about 50% thrust until both engines are stabilized
(N1 stays constant at around 50%) while still holding the brakes.

> - Push Sidestick forward half the way to put pressure on the front gear<br/>

Reviewer 2: ```suggestion
- Push sidestick forward half the way to put pressure on the front gear<br/>
```

        shown in blue on the PFD's speed tape.

        **V~1~**: The speed beyond which takeoff should no longer be aborted. V~1~ is depicted as a "1" next to the speedband in the PFD.
        
        **V~R~**: Rotation speed. The speed at which the pilot begins to apply control inputs to cause the aircraft nose to pitch up, after which it will leave the ground. V~R~ is depicted as a cyan circle next to the speedband in the PFD.
        
        **V~2~**: Takeoff safety speed. The speed at which the aircraft may safely climb with one engine inoperative. V~2~ is depicted by a magenta triangle next to the speedband in the PFD.
        
        On a long enough runway V~1~ (depicted by "1") and V~R~ (depicted by "o") are often very close together and can't be clearly distinguished on the PFD speed tape.

- At about 80 knots slowly release the forward pressure on the sidestick until about 100 knots when the sidestick should be in neutral position.

- The throttle hand remains on the thrust levers until reaching V~1~ to be
  able to quickly abort the start. Remove the hand from the thrust levers
  at V~1~ to avoid accidentally aborting after V~1~.

> - At V~R~ gently pull back on the Sidestick and try to get a 10-15° climb angle.

Reviewer 2: ```suggestion
- At V~R~ gently pull back on the sidestick and try to get a 10-15° climb angle.
```

Reviewer 3: Shouldn't this reference a pitch attitude rather than a climb angle?  How would one determine the climb angle in SRS mode? The FCTM refers to a smooth positive backward stick movement rather than gently pulling back on the stick. Maybe that's a better description? It also says to aim for a rotation rate (pitch rate) of 3 deg/sec and once airborne to follow the flight director rather than a specific pitch attitude target. 10-15 seems a bit shallow to me. If you are going to give such a target, maybe just say 15 (or 15-18). One advantage to using 15 would be to then use Shomas' method of hitting the 3 deg/sec target, by suggesting to get to 15 deg pitch attitude in 5 seconds (for example, counting one-one thousand, two-one-thousand, etc. and hitting 15 deg at five-one-thousand.  (Just a thought.)

Reviewer: @donstim great feedback. Will change this as suggested.

In the A320 (and most Airbus models) we will not touch the thrust levers again
before final approach and landing (under normal flight conditions).

The aircraft will now climb to the altitude selected in the FCU (in our
case 5.000ft).

**Activate the __Autopilot__ at this point by pressing the AP1 button on the FCU.**

![Autopilot and Autothurst buttons](../assets/takeoff-climb-cruise/AP1.png "Autopilot and Autothurst buttons"){width=280}

The FMA now shows AP1 in white in the upper right corner.

!!!info "FCU Autopilot Controls"
    The FCU (Flight Control Unit) shows three important values:

    - SPD "---" : means the __Autopilot__ is in Managed Speed mode (e.g. 250knots
      <1000ft, 290 above). If we pull the SPD knob we can select a speed which
      the __Autopilot__ will then apply.
    - HDG "---" : means the lateral navigation is in Managed HDG Mode and the
      __Autopilot__ follows the planned route. Dialing the HDG knob will let us
      select a heading and by pulling the knob
      we tell the __Autopilot__ to fly this heading (Selected Heading Mode).<br/>
    - ALT "5000" : means the selected altitude is 5000ft<br/>

> **When reaching S-speed retract flaps (assuming we took off with FLAPS 1).**<br/>

Reviewer 3: Shouldn't need the "(assuming we took of with Flaps 1+F)."  Can't take off with Flaps Up, and whether you use CONF 1+F (FCOM never refers to it as "FLAPS 1+F" ), 2, or 3, when you get to S speed you select flaps up. When taking off with CONF 2 or 3, you select CONF 1 (which FCOM does refer to as FLAPS 1).

> **When reaching S-speed retract flaps (assuming we took off with FLAPS 1).**<br/>

Reviewer: Removed the "assuming". And I agree it is either FLAPS 1 or CONF-1+F - don't think it should say FLAPS 1+F. 
As FLAPS is easier for the beginner I chose to use both when CONF is mentioned.  
Also changed lowercase flaps x to uppercase FLAPS x as it is a specific setting.

- *After take off checklist* is completed.
- ATC has given us clearance for further climb.

**Dial the newly cleared altitude into the FCU. (e.g., 12000ft)**

![Entering the new altitude and pushing knob for Managed Altitude Mode](../assets/takeoff-climb-cruise/Altitude_setting12000.png "Entering the new altitude and pushing knob for Managed Altitude Mode")

The aircraft will now continue climbing while managing thrust and pitch level.
The __Autopilot__ ensures that the aircraft stays at the Selected or Managed
Speed setting and climbs to the new altitude while managing thrust automatically.

The PFD's FMA now shows:<br/>
![FMA during climb](../assets/takeoff-climb-cruise/FMA_THR_CLB.png "FMA during climb")

Thrust level is "THR CLB", vertical mode is "CLB" (ALT mode armed), lateral
mode is NAV.

Typically, the climb to the flight plan's cruise level (e.g., FL210) happens
in several steps (step climbs). Each to be instructed and cleared by ATC.

**Passing 10,000ft**<br/>
Turn off landing lights and when the aircraft is stable (weather, no turn,
etc.) you can turn off the seatbelt signs. The aircraft will now accelerate
to CLB speed (defined in MCDU PERF CLB page).

> **Repeat the climb process above until cruise level (e.g. FL210) is reached.**
> 
> ![ECAM PERF CRZ page at cruise level](../assets/takeoff-climb-cruise/ECAM_PERF_CRZ_cruise_lvl.png "ECAM PERF CRZ page at cruise level"){ align=left width=320 }
> 
> ![PFD at cruise level](../assets/takeoff-climb-cruise/Cruise_lvl_pfd.png "PFD at cruise level"){ align=right width=312 }
> 
> <p/>
> 
> This concludes the *Climb*.

Reviewer 2: This seems entirely broken

Current Iteration:
![image](https://user-images.githubusercontent.com/1619968/123529829-e1aef200-d726-11eb-9f08-e6bab08d46c2.png)

I came up with a workaround but if you find something different you can do that as well. 

```suggestion
**Repeat the climb process above until cruise level (e.g. FL210) is reached.**

=== "MCDU + PFD"
    ![ECAM PERF CRZ page at cruise level](../assets/takeoff-climb-cruise/ECAM_PERF_CRZ_cruise_lvl.png "ECAM PERF CRZ page at cruise level"){ align=left width=320 }

    ![PFD at cruise level](../assets/takeoff-climb-cruise/Cruise_lvl_pfd.png "PFD at cruise level"){ align=right width=312 }

This concludes the *Climb*.
```
![image](https://user-images.githubusercontent.com/1619968/123529852-14f18100-d727-11eb-87fa-33596c9978b8.png)

Reviewer: Yea, it looksed ok on my screen but obviously there is difference with yours. Maybe broawser. But your solution is neat. I thought about having something like !!!info for images to also have image descriptions but couldn’t find anything.

---

### 5. Cruise
**Situation:**

> - Aircraft has leveled of at planned cruise level.

Reviewer 2: ```suggestion
- Aircraft has leveled off at planned cruise level.
```

- Speed is cruise speed as per ECAM PERF CRZ page.
- __Autopilot__ is ON.
- Speed is in Managed Mode.

This is usually the quietest time of the flight.
It allows time to double-check the systems by going through all ECAM pages, etc.

> Regular ATC frequency changes with altitude and position checkins are common.

Reviewer 2: ```suggestion
Regular ATC frequency changes with altitude and position check-ins are common.
```
It's either that or check ins. I'm 90% sure it's with the -

Reviewer: https://writingexplained.org/check-in-or-checkin ==> it's "check-ins" as it is used as a noun.

Here are some **typical activities** which might happen during cruise mostly on
request from ATC or other circumstances like weather, traffic, etc.

- **Altitude change (also called flight level change)**<br/>
  Like before during climb set your new altitude in the FCU and push the

>   ALT knob. The aircraft will descent or climb to the new altitude

Reviewer 2: ```suggestion
  ALT knob. The aircraft will descend or climb to the new altitude
```

  automatically.

- **Course change with Selected Heading** (given or cleared by ATC)<br/>
  Dial heading knob to the desired heading and pull knob for Selected
  Heading Mode. The aircraft will automatically change course to the new
  heading. If you want the aircraft to follow the planned route again
  you can push the knob for Managed Heading Mode.

- **Direct course to a waypoint (DIR TO)** <br/>
  ATC regularly instructs us to go "direct to (waypoint) XYZ".
  Use the ECAM DIR page to select the waypoint from the plight plan's
  list of waypoints. In rare cases it is a waypoint not in the current
  flight plan then you can enter a new waypoint in the MCDU and put it

>   into the upper left entry field. Select DIRECT* on the rght bottom to

Reviewer 2: ```suggestion
  into the upper left entry field. Select DIRECT* on the bottom right to
```

  execute the change.

    ![ECAM direct to](../assets/takeoff-climb-cruise/Dir_to.png "ECAM direct to"){ width=360 }

- **ATC requests specific speed **<br/>
  Sometimes ATC requests a specific speed to keep separation between
  aircraft. Pull the speed knob to switch to Selected Speed Mode. The
  current speed will be preselected. Dial to the desired speed. The aircraft

>   will immediately begin to target the new speed by either incresing

Reviewer 2: ```suggestion
  will immediately begin to target the new speed by either increasing
```
